detox/local-cli/utils/frameworkUtils.js

```diff
@@ -0,0 +1,78 @@
+const os = require('os');
+const path = require('path');
+
+const exec = require('child-process-promise').exec;
+const fs = require('fs-extra');
+const _ = require('lodash');
+
+const detox = require('../../internals');
+
+const frameworkBuildScript = '../../scripts/build_local_framework.ios.sh';
+const xcuitestBuildScript = '../../scripts/build_local_xcuitest.ios.sh';
+
+const getFrameworkPath = _.once(() => path.join(os.homedir(), '/Library/Detox/ios/framework'));
+const getXcuitestPath = _.once(() => path.join(os.homedir(), '/Library/Detox/ios/xcuitest-runner'));
+
+function shouldSkipExecution() {
+  if (os.platform() !== 'darwin') {
+    detox.log.info('The command is supported only on macOS, skipping the execution.');
+    return true;
+  }
+
+  return false;
+}
+
+async function execBuildScript(targetPath, scriptPath, descriptor) {
+  detox.log.info(`Building ${descriptor} cache at ${targetPath}..`);
+
+  try {
+    const result = await exec(path.join(__dirname, scriptPath), { capture: ['stdout', 'stderr'] });
+    detox.log.info(result.stdout);
+  } catch (error) {
+    detox.log.error(`Error while building ${descriptor}: ${error.stderr}`);
```

Comment:
I'm unsure if the error logs will always be in `stderr`. Could you make sure that both stdout/stderr get included?

Also, is there a good reason why you aim to **capture** stdout/stderr? Why not just let it flow through? I mean `stdio: 'inherit`? 

In this mode, detox doesn't create any log files, so we're not losing any precious info, be it good or not.

Reply:
migrated to `stdio: 'inherit'`